ui/main_window.cpp

```diff
@@ -1172,6 +1183,20 @@ bool MainWindow::LoadFile(const std::string &file_name, bool is_temp_file)
             m_gpu_timing_model->OnGpuTimingResultsGenerated("");
             qDebug() << "Failed to find gpu timing data";
         }
+
+        // Check if there is an existing screenshot
+        qDebug() << "Attempting to load screenshot from: " << screenshot_file_path.string().c_str();
```

Comment:
you will have some problems of merging this code block, maybe you need to verify with @hysw, with [pr435](https://github.com/google/dive/pull/435), MainWindow::LoadFile supports async load now